css/templates/travel/elements/_block-list.css

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2016 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+/*
+ * block-list
+ *
+ * Style an <amp-list> element to display itself and its children as block
+ * elements, restoring its intrinsic height.
+ */
+.travel-block-list {
+  min-height: 1px;
+  min-width: 1px;
+  position: static;
+}
+
+.travel-block-list [role="list"] {
+  position: static;
+  height: auto;
+  width: auto;
+}
+
+/*
+  Add an nbsp character to the block-list element to ensure prevent it
+  from shrinking to 0px X 0px when otherwise empty which would make AMP think
+  the element was not visible in the viewport and would prevent the list
+  request from firing.
+ */
+.travel-block-list::after {
+  content: "\00a0";
```

Comment:
just adding a space here should have similar effect?

Reply:
What do you mean a space?